### ISSUE TITLE
Source archives on website don't contain ddnet-libs

### ISSUE THREAD

**Issue by Robyt3** [ddnet/ddnet#6323]
The folder `ddnet-libs` is empty when downloading the source from https://ddnet.org/downloads/, which makes the cmake configuration fail due to missing libraries when trying to use the bundled libraries. Shouldn't the bundled libraries be included in source archives?

**Comment by def-** [comment]
I think they shouldn't be included since they are pretty large and useless for most people who will use the exact version downloads, namely distributions which build the code from it.

**Comment by Chairn** [comment]
I agree that they should be included, but we can either add a warning on the website with associated separated ddnet-lib download or bundle both in a second download option

**Comment by sjrc6** [comment]
What purpose does the source archive serve if you can't build them? Building the release versions from the git repo is hard (impossible?) because the releases are constructed outside the main branch. 

Also I disagree that they are "not part of the source code" because no where in the folder does it tell you which library versions it was built with, so they could be literally anything.

**Comment by heinrich5991** [comment]
It's expected that you have already built the libraries yourself. It's generally the latest version, these libraries rarely make breaking changes. It's rare to ship pre-built libraries in source archives for open-source software.

What problem do you want to solve? What OS are you on? What are you trying to build?
